sig-service-catalog/charter.md

```diff
@@ -53,12 +53,12 @@ This SIG’s main goals are:
 ### Out of scope
 
 The following, non-exhaustive, items are out of scope:
-- Operation of OSBAPI Service Brokers.
+- Operation of any particular service
```

Comment:
A mention of [Jenkins](https://service-catalog-jenkins.appspot.com/) server above runs end-to-end tests on a live cluster. Is that an exception to this out-of-scope item?

Reply:
Good catch, @baijum 
The e2e tests are now running in prow, and the Jenkins seems to be dysfunctional.